### PR TITLE
Pull sc calls into controller (dependent on 255)

### PR DESCRIPTION
**Pull request by willKim19**
A lot of calls to the token contracts themselves are abstracted into external functions, but this abstraction is unnecessary and requires a lot of jumping across files to get the full picture. 

This PR pulls those calls directly into the controller functions, with the ultimate goal of these controller functions being as standalone as possible.

### CODE REVIEW

**Comment by iAmMichaelConnor** [comment]
@willKim19 I'm not sure about this change. Having a separate file acting as a 'gateway' to the smart contract was an intentional separation. All calls / tx's to the smart contract happen in a single file, and that file does nothing else.

I think it's worth a couple of rounds of discussion to decide on the best approach here.

**Comment by willKim19** [comment]
My thought was that the abstraction (i.e., the zkp.mint() function call) and the contents of the function themselves are about the same in terms of complexity, so it would make sense just to bring it in.

Could you explain the reasoning behind the separation? It doesn't seem like any other files/functions use these functions, which is the typical reason why you'd want to abstract the function out like this.

File: zkp/src/f-token-controller.js [hunk 1]
```diff
@@ -268,29 +270,22 @@ async function mint(amount, ownerPublicKey, salt, vkId, blockchainOptions) {
...
-    proof,
-    inputs,
-    vkId,
-    amount,
-    commitment,
-    account,
-    fTokenShieldInstance,
-  );
+  // Mint the commitment
+  const txReceipt = await fTokenShieldInstance.mint(proof, inputs, vkId, amount, commitment, {
```
Comment (ChaitanyaKonda) [inline zkp/src/f-token-controller.js]: Would it be possible to move the console.logs from f-token-zkp.js mint() to here?
The console logging helps a lot in debugging problems that pop from external users as well as while building new features. And please retain all the console logs everywhere too.

**Comment by iAmMichaelConnor** [comment]
@willKim19 you, @Westlad & @ChaitanyaKonda have convinced me this PR is a good idea.

File: zkp/src/f-token-zkp.js [hunk 2]
```diff
@@ -221,9 +83,6 @@ async function checkCorrectness(C, pk, S, z, zIndex, fTokenShield) {
 }
 
 export default {
-  mint,
-  transfer,
-  burn,
   registerVk,
```
Comment (ChaitanyaKonda) [inline zkp/src/f-token-zkp.js]: registerVk,
  registerVerifierContract,
  setVkIds,

Can the same as above be done for these functions too? To add them directly to vk-controller.js.

Comment (willKim19) [reply zkp/src/f-token-zkp.js]: I'll do that in a separate PR

File: zkp/src/f-token-controller.js (hunk 1 above)
Comment (willKim19) [reply zkp/src/f-token-controller.js]: I've brought the console.logs back in. I still think that there's probably some redundant console logs that might not be as useful, but maybe we can have a future discussion about them.

**Pull request merged**